src/schema/datatypes/meg.yaml

```diff
@@ -0,0 +1,42 @@
+- suffixes:
+    - meg
+  extensions:
+    - 4d/
```

Comment:
Now that I look at it, it looks like the [4d format](https://bids-specification.readthedocs.io/en/stable/99-appendices/06-meg-file-formats.html#bti4d-neuroimaging) just has a folder with no suffix. Here's what it looks like:

```
sub-control01/
    ses-001/
        sub-control01_ses-001_scans.tsv
        meg/
            sub-control01_ses-001_coordsystem.json
            sub-control01_ses-001_headshape.pos
            sub-control01_ses-001_task-rest_run-01_meg/
                config
                hs_file
                e,rfhp1.0Hz.COH
                c,rfDC
            sub-control01_ses-001_task-rest_run-01_meg.json
            sub-control01_ses-001_task-rest_run-01_channels.tsv
```

I guess I should just put a `/` there, instead of `4d/` or `.4d/`?

Reply:
yeah, this format is the biggest pain ever ... we should ask some MEG people what to do (in particular, Robert O. once he's back from vacation)

Can we have `/` with a comment that this is for BTi/4D data?

Reply:
Done in 4e291e2, but I'll hold off on resolving this in case further discussion is needed.